lib/mayaUsd/ufe/UsdTRSUndoableCommandBase.cpp

```diff
@@ -69,6 +86,12 @@ void UsdTRSUndoableCommandBase<V>::redoImp()
     }
 
     perform(fNewValue[0], fNewValue[1], fNewValue[2]);
+
```

Comment:
As per private conversation, this should go before line 78, at the beginning of redoImp().

Reply:
@ppt-adsk  addressed in 07014386de044e301769584c08bdb7094844b30f